app/javascript/components/pages/Donate.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useMemo } from 'react';
+import propTypes from 'prop-types';
+import SharedLayout from 'components/layout/SharedLayout';
+import Banner from 'components/Banner';
+import Button from 'components/Button';
+import PageTitleWithContainer from 'components/PageTitleWithContainer';
+import { testDonationAmounts, productionDonationAmounts } from 'datasources';
+
+import 'stylesheets/page';
+import 'stylesheets/donate';
+
+const Donate = () => {
+    const [selectedPrice, setSelectedPrice] = useState(100);
+
+    return (
+        <SharedLayout>
+            <PageTitleWithContainer text="Donate" />
+            <div className="flex flex-row p-12 justify-center flex-wrap">
+                <div className="flex flex-col mb-20 md:mb-0 md:mr-20">
+                    <h2 className="text-2xl font-medium mb-8">Choose a donation amount</h2>
+                    <DonationAmounts
+                        selectedPrice={selectedPrice}
+                        setSelectedPrice={setSelectedPrice}
+                    />
+                </div>
+                <div className="max-w-[25rem]">
+                    <h2 className="text-2xl font-medium mb-8">Why give to WNB.rb?</h2>
+                    <p className="mb-5">
+                        Donating to WNB.rb enables us to invest more resources in supporting women
+                        and non-binary people in the Ruby community.
+                    </p>
+                    <p>
+                        Here are some of the initiatives we plan to launch in 2022 with your help:
+                    </p>
+                    <ul className="list-disc ml-10">
+                        <li>Community fund for educational materials</li>
+                        <li>Full scholarships to RubyConf and RailsConf</li>
+                        <li>Open source development stipends</li>
+                    </ul>
+                </div>
+            </div>
+            <OtherAmountBanner />
+        </SharedLayout>
+    );
+};
+
+const DonationAmounts = ({ selectedPrice, setSelectedPrice }) => {
+    const prices = useMemo(() => {
+        return process.env.NODE_ENV === 'development'
+            ? testDonationAmounts
+            : productionDonationAmounts;
+    }, []);
```

Comment:
@stephanieminn I'd like your feedback on this pattern -- there are two sets of donation links, test links and real links. The test links prevent us from polluting our real payment data when making test payments. I've set this up by creating two JavaScript objects and here I'm using the env to determine which object to use as the prices. Does this... make sense?